src/app/home/components/home/home.component.html

```diff
@@ -0,0 +1,47 @@
+<header-component></header-component>
+
+<div class="cover">
+    <p>
+        Dany y James
+        <span></span>
```

Comment:
If you need to draw a line you could use a div with class...
also you could avoid use the close tag with <span/>